polaris-react/src/components/ActionList/components/Item/Item.tsx

```diff
@@ -80,11 +81,23 @@ export function Item({
     </span>
   );
 
-  const suffixMarkup = suffix && (
-    <Box>
-      <span className={styles.Suffix}>{suffix}</span>
-    </Box>
-  );
+  let suffixMarkup: React.ReactNode | null = null;
+
+  if (active) {
+    suffixMarkup = (
+      <Box>
+        <span className={styles.Suffix}>
+          <Icon source={TickSmallMinor} />
+        </span>
+      </Box>
+    );
+  } else if (suffix) {
+    suffixMarkup = suffix && (
+      <Box>
+        <span className={styles.Suffix}>{suffix}</span>
+      </Box>
+    );
+  }
```

Comment:
I have tentatively made the suffix to always be `TickSmallMinor`` when active is `true` following @chloerice's [suggestion](https://shopify.slack.com/archives/C4Y8N30KD/p1680722435876329?thread_ts=1680622215.701159&cid=C4Y8N30KD). 
Will update if the consensus changes.